stdlib/public/core/Unicode.swift

```diff
@@ -140,260 +227,61 @@ public struct UTF8 : UnicodeCodec {
     let bit1 = (lut1 >> index) & 1
 
     switch (bit1, bit0) {
-    case (0, 0): // 2-byte sequence.
+    case (0, 0): // 2-byte sequence, [ XXX XXX CU1 CU0 ].
```

Comment:
`[ XXX XXX CU1 CU0 ]` doesn't make sense to me.  Everywhere else we've been using X's for bits.  What do they mean here?